### PR TITLE
Exponential Linear Unit activation is added

### PR DESCRIPTION
**Pull request by Mitra-babu**
### Describe your change:

The ELU activation is added.
 Mathematically, f(x) = x, x>0 else (alpha * (e^x -1)), x<=0, alpha >=0

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Mitra-babu** [comment]
@cclauss  please review this and suggest in case any changes need.

File: maths/elu_activation.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+def elu_activation(vector: np.ndarray, alpha: float) -> np.ndarray:
+    """
+         Implements the ELU activation function.
+         Parameters:
+             vector: np.array
+             alpha: float
+         return:
+         elu (np.array): The input numpy array after applying elu.
```
Comment (cclauss) [inline maths/elu_activation.py:21]: Please do not repeat the data types from the function signature.  The problem with doing so is that the reader becomes confused if one changes and the other does not.  Either describe what these variables are for or remove these comments.

File: maths/elu_activation.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+input and then applies the ELU function to each element of the vector.
+
+Script inspired from its corresponding Wikipedia article
+https://en.wikipedia.org/wiki/Rectifier_(neural_networks)
+"""
+
+import numpy as np
+
+
+def elu_activation(vector: np.ndarray, alpha: float) -> np.ndarray:
```
Comment (cclauss) [inline maths/elu_activation.py:14]: This repo resists using acronyms...
```suggestion
def exponential_linear_unit(vector: np.ndarray, alpha: float) -> np.ndarray:
```

**Review by cclauss — COMMENTED** [review]
This would probably be better placed in our `neural_network` directory.

**Pull request closed without merging**